src/xcode/ENA/ENA/Source/Scenes/DataDonation/DataDonationViewController.swift

```diff
@@ -26,12 +28,25 @@ class DataDonationViewController: DynamicTableViewController, DeltaOnboardingVie
 
 	override func viewDidLoad() {
 		super.viewDidLoad()
-		parent?.navigationItem.title = AppStrings.DataDonation.Info.title
-		parent?.navigationItem.largeTitleDisplayMode = .never
-		parent?.navigationController?.navigationBar.prefersLargeTitles = true
+		var effectiveNavigationItem: UINavigationItem?
```

Comment:
Can this be a `let`?

Reply:
Yes, it's possible (... but now the comment seems maybe a bit too long / out of balance ...)

Note: actually, I don't like how the view controllers tamper with the navigation items of their parent - it breaks encapsulation. I have submitted a github ticket with a suggestion to improve this (see #3474), and I plan to submit a PR with a solution for this.

Reply:
I would propose to define it as a let until you request the other PR?

Reply:
@pascal-brause sure, done. Please see my newest commit on this branch.
(By the way, the other PR is #3483).